docs/Core.rst

```diff
@@ -1,152 +1,197 @@
-a hotkey name (with ``n``).
+Only bindings with a ``context`` tag that either matches the current context fully,
+or is a prefix ending at a ``/`` boundary would be considered for execution, i.e.
+when in context ``foo/bar/baz``, keybindings restricted to any of ``@foo/bar/baz``,
+``@foo/bar``, ``@foo`` or none will be active.
 
-A new and more flexible way is the keybinding command in the dfhack console.
-However, bindings created this way are not automatically remembered between runs
-of the game, so it becomes necessary to use the dfhack.init file to ensure that
-they are re-created every time it is loaded.
+Multiple contexts can be specified by separating them with a
+pipe (``|``) - for example, ``@foo|bar|baz/foo`` would match
+anything under ``@foo``, ``@bar``, or ``@baz/foo``.
 
 Interactive commands like `liquids` cannot be used as hotkeys.
 
-Many commands come from plugins, which are stored in ``hack/plugins/``
-and must be compiled with the same version of DFHack.  Others come
-from scripts, which are stored in ``hack/scripts``.  Scripts are much
-more flexible about versions, and easier to distribute - so most third-party
-DFHack addons are scripts.
 
-Something doesn't work, help!
-=============================
-First, don't panic :)
+.. _kill-lua:
 
-Second, dfhack keeps a few log files in DF's folder (``stderr.log`` and
-``stdout.log``). Looking at these might help you solve the problem.
-If it doesn't, you can ask for help in the forum thread or on IRC.
+kill-lua
+--------
+Stops any currently-running Lua scripts. By default, scripts can
+only be interrupted every 256 instructions. Use ``kill-lua force``
+to interrupt the next instruction.
 
-If you found a bug, you can report it in `the Bay12 DFHack thread`_,
-`the issues tracker on github <https://github.com/DFHack/dfhack/issues>`_,
-or visit `the #dfhack IRC channel on freenode
-<https://webchat.freenode.net/?channels=dfhack>`_.
 
-=============
-Core Commands
-=============
-DFHack command syntax consists of a command name, followed by arguments separated
-by whitespace. To include whitespace in an argument, quote it in double quotes.
-To include a double quote character, use ``\"`` inside double quotes.
+.. _load:
+.. _unload:
+.. _reload:
 
-If the first non-whitespace character of a line is ``#``, the line is treated
-as a comment, i.e. a silent no-op command.
+load
+----
+``load``, ``unload``, and ``reload`` control whether a plugin is loaded
+into memory - note that plugins are loaded but disabled unless you do
+something.  Usage::
 
-When reading commands from dfhack.init or with the ``script`` command, if the final
-character on a line is a backslash then the next uncommented line is considered a
-continuation of that line, with the backslash deleted.  Commented lines are skipped,
-so it is possible to comment out parts of a command with the ``#`` character.
+    load|unload|reload PLUGIN|all
```

Comment:
Has this changed since 40.24-r3?  On the version I'm testing it's definitely `unload all`, and `-all` gives an error that it can't find that plugin.